crates/rpc/rpc-builder/tests/it/startup.rs

```diff
@@ -105,7 +117,7 @@ async fn test_launch_same_port_different_cors() {
         EthApiBuild::build,
     );
     let addr = test_address();
-    let res = RpcServerConfig::ws(Default::default())
+    let res = RpcServerConfig::<Identity>::ws(Default::default())
```

Comment:
this shouldn't be necessary if we put the init functions in another impl block that doesn't require this generic